### ISSUE TITLE
Add documentation for contributors | CONTRIBUTING.md

### ISSUE THREAD

**Issue by Snailedlt** [SplitScreen-Me/splitscreenme-nucleus#22]
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors
Add some contributing guidelines, and reference them in the README.md

**Comment by RangerTJ** [comment]
Howdy!

Just wanted to give a heads-up, I'm looking to have a PR with a CONTRIBUTIONS.md file ready for review in the near future. The general outline of the file looks like:

Part 1: Brief intro and link to handler development FAQ
Part 2: Bugs / Issues / link to issues page (includes security issue note)
Part 3: Developer Guide (how to fork, PR, branch guide, compiling, etc.)
Part 3b: Code Style Guide (barebones for now; can be expanded or scrapped based on preferences)
Part 4: Donations (points users to donations sidebar on the main page)

**WIP Draft** (has some notable gaps/assumptions in its current, rough state that I'm looking to patch up)
[CONTRIBUTIONS-draft.md](https://github.com/SplitScreen-Me/splitscreenme-nucleus/files/15358648/CONTRIBUTIONS-draft.md)

Comments and feedback are appreciated! _Especially_ if there are gaps in guide content. I've attached a draft I've been working on as a starting point. It currently makes assumptions about code style, security issues, and the PR process/branching, and I'm reasonably sure most of these sections will need to be edited in some capacity. I'm also looking to add a brief/high-level compiling tutorial so it's more accessible for folks to fork and test their changes if they're attempting any code contributions.

I want to emphasize the draft I've attached is a _starting_ point, heh. If it makes more sense, I could also potentially do a slimmer initial PR for the document that has less info if the project doesn't want to commit to specific policies just yet. I wanted to post this here, in case any others active in the project can think of anything else that I might be missing (or anything that I might want to change/remove/restructure).

Thanks!
